lxc/action.go

```diff
@@ -245,6 +249,16 @@ func (c *cmdAction) doAction(action string, conf *config.Config, nameArg string)
 		return err
 	}
 
+	if action == "stop" {
+		// Handle console attach
+		if c.flagConsole != "" {
```

Comment:
Those two if statements can be turned into one.

Reply:
Done